### PR TITLE
Update what-else-do-you-need-to-know.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
fix-incorrect-rendering-of-pq-and-rq-for-what-else-you-need-to-know-appsec-insight

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/security/appsec/security-intro/what-else-do-you-need-to-know.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/security/appsec/security-intro/what-else-do-you-need-to-know.md?ref=627461602c563f6085f75c94505f8b336ed64aef)

File: security/appsec/security-intro/what-else-do-you-need-to-know.md [hunk 1]
```diff
@@ -41,10 +41,14 @@ Before diving into this skill, you should make sure you have a strong foundation
...
-Operating Systems  
-How memory in a computer works, and where data is located at any given time  
+**At least one scripting language (even bash, check out our other topics)**
+
+???
+
+**Operating Systems**
+
+**How memory in a computer works, 
+and where data is located at any given time**
```
Comment (kapnobatai137) [inline security/appsec/security-intro/what-else-do-you-need-to-know.md:51]: I think these should go on the same line
```suggestion
**How memory in a computer works, and where data is located at any given time**
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

File: security/appsec/security-intro/what-else-do-you-need-to-know.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply security/appsec/security-intro/what-else-do-you-need-to-know.md]: Ok, I applied your suggestion

**Review by kapnobatai137 — DISMISSED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**